### PR TITLE
MISP-1793: chipageddon support old and new wifi modules (S5R2/S3)

### PR DESCRIPTION
**Pull request by bryanfoley**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "linux"]
 	path = linux
-	url = https://github.com/Ultimaker/linux
+	url = https://github.com/Ultimaker/linux.git
```
Comment (robinmdh) [inline .gitmodules:3]: the addition of .git seems odd to me....

**Review by robinmdh — DISMISSED** [review]

File: .gitmodules (hunk 1 above)
Comment (bryanfoley) [reply .gitmodules:3]: That;s the url, submodule usually just appends the .git underwater, but you can also provide it.

**Review by CoenSchalkwijk — APPROVED** [review]

**Review by bryanfoley — COMMENTED** [review]
I opened the PR so I can't officially approve but I approve in spirit :+1:

**Review by shanx — APPROVED** [review]

**Review by robinmdh — APPROVED** [review]
LGTM

**Review by robinmdh — APPROVED** [review]
LGTM

**Pull request merged**